.eslintrc.json

```diff
@@ -9,6 +9,7 @@
             "experimentalObjectRestSpread": true,
             "jsx": true
         },
+        "ecmaVersion": 2017,
```

Comment:
Enable ES6 [async/await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await) syntax, which is currently supported by all browsers.